scripts/src/main/resources/scripts/command/kubernetes

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# $1: optional setup
+function doSetup() {
+  if doIsKubernetesInstalled
+  then
+    doEcho "Kubernetes is already installed at $(command -v kubectl)"
+    exit 0
+  else
+    if doIsBatch
+    then
+      doFail "Interactive installation is required, cannot proceed in batch mode. Please rerun without batch option."
+    fi
+
+    if doIsWindows
+    then
+      if doIsWSLEnabled
+      then
+        # Check if Kubernetes is already installed on WSL
+        if doIsKubernetesInstalledOnWSL 
+        then
+          echo Kubernetes is already installed on WSL and ready to use. devonfw-ide did not make any changes.
+          wsl kubectl version
+          exit 0
+        else 
+          echo Kubernetes not installed on WSL.
+          # Install Kubernetes on WSL
+          wsl bash ../scripts/setup-kubernetes
+        fi        
+      fi
+    else
+      echo "Sorry, kubernetes-installation support is not yet implemented for your OS. Please install manually or help devonfw-ide to support it for your OS by contributing a pull-request."
+      exit 0
+    fi
+  fi  
+}
+
+
+# Call Kubernetes with specified arguments
+function doRun() {
+  if doIsWindows
```

Comment:
why should we limit this to windows?
If `kubectl` is available on linux or MacOS we can make `devon kubernetes «args»` delegate to `kubectl «args»` without errors.
Why not simply reusing `doSetup` as we do in other commandlets? (we specify an argument to `doSetup` so we can distinguish if the user wants to setup explicitly or implicitly).